travis/test_server.py

```diff
@@ -393,7 +391,10 @@ def main(argv=None):
             test_loghandler = ['openerp.tools.yaml_import:DEBUG',
                                'openerp.models.schema:DEBUG']
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
-    server_path = get_server_path(odoo_full, odoo_version, odoo_branch, travis_home)
+    _version = odoo_version
```

Comment:
@moylop260 
What do you think of this change?
I did not want to modify the odoo_version variable because it is used in several parts